src/engraving/dom/dynamic.cpp

```diff
@@ -519,11 +636,56 @@ mu::RectF Dynamic::drag(EditData& ed)
             PointF pos2(canvasPos());
             const PointF newOffset = pos1 - pos2;
             setOffset(newOffset);
+            setOffsetChanged(true);
             ElementEditDataPtr eed = ed.getData(this);
             eed->initOffset += newOffset - oldOffset;
         }
     }
-    return f;
+
+    EngravingItem::editDrag(ed);
+}
+
+//---------------------------------------------------------
+//   endEdit
```

Comment:
I would recommend not using these comments. They are from the MS3 era and only clutter up the code